projects/plugins/jetpack/extensions/blocks/subscriptions/settings.scss

```diff
@@ -18,6 +18,10 @@
 .edit-post-post-visibility {
 	&__notice {
 		margin: 0px;
+		// General notice
+		&.is-info {
+			background: #e9eff5; // Found using color picker on figma.
```

Comment:
I looked through the all the jetpack code to see if this background color existed somewhere else, or if the background was getting applied to notices and I didn't find any.  That's why I just put in the color straight from the figma file.

Reply:
Hi @n3f !

Sorry I missed this comment.

I've used Blue / Blue 0 (`studio-blue-0`), but maybe de HEX code is slightly different. Do you have that studio color?

If you don't have it, can you please share with me the blue hex you have, so I can find what can we use for this particular case?

As a side note: the `notice` component on Gutenberg has some accessibility issues. I plan to open an issue in the Gutenberg repo so we can address it (and this would include add the blue background as well).

Reply:
I am not sure hard-coding the background is a good idea: people that are using different theme could have some issues: https://a8c.slack.com/archives/C052XEUUBL4/p1683645627179279?thread_ts=1683639652.285329&cid=C052XEUUBL4

Reply:
I discussed with Brent to use the SCSS lighten with:

```
lighten(var(--wp-components-color-accent,var(--wp-admin-theme-color,#007CBA)), 0.5) //adapt to fit Figma
```